server/routerlicious/packages/lambdas/src/test/types/validateServerLambdasPrevious.generated.ts

```diff
@@ -29,7 +29,7 @@ type TypeOnly<T> = T extends number
 declare function get_old_ClassDeclaration_BroadcasterLambda():
     TypeOnly<old.BroadcasterLambda>;
 declare function use_current_ClassDeclaration_BroadcasterLambda(
-    use: TypeOnly<current.BroadcasterLambda>): void;
+    use: TypeOnly<current.BroadcasterLambda>);
```

Comment:
I saw a comment in another PR of how it was nice that something added the return types... and here we lost them? 😄